packages/file_selector/file_selector_ios/ios/file_selector_ios/Sources/file_selector_ios/FFSFileSelectorPlugin.m

```diff
@@ -3,8 +3,9 @@
 // found in the LICENSE file.
 
 #import "FFSFileSelectorPlugin.h"
+
+#import "./include/file_selector_ios/messages.g.h"
```

Comment:
Why was adding the relative path to the `#import` necessary, if it's in the header search path?

Reply:
So for most of the headers, include paths is sufficient. However, `messages.g.h` is a problem when there's multiple plugins that also have a `messages.g.h`. Seems like it can't resolve which to use. This appeared to also be a problem with CocoaPods, more context here: https://github.com/flutter/flutter/issues/147587#issuecomment-2086461349

Reply:
Should this instead wait until https://github.com/flutter/packages/pull/6600 is done then?

Reply:
I'm fine with manually doing include path fixup in advance of the fix landing, since we know the generator will catch up with it and it's not changing anything that could have runtime effects.